### ISSUE TITLE
fits microservice uri default fails in crayfits tag 1.0.0-alpha-14

### ISSUE THREAD

**Issue by jefferya** [Islandora-Devops/isle-buildkit#201]
When using the isle-buildkit version with the removed nginx proxies (e.g., tag 1.0.0-alpha-14) in conjunction with https://github.com/Islandora-Devops/isle-dc/pull/222, ingestion of a video fails to product a fits media file. 

Inspecting the crayfits container:
```
bash-5.1# cat /var/log/islandora/fits.log
[2022-03-23 22:35:41] islandora_fits.ERROR: ERROR ["cURL error 7: Failed to connect to fits port 80 after 0 ms: Connection refused (see https://curl.haxx.se/libcurl/c/libcurl-errors.html)"] []
```

and from the docker-compose crayfits log
```
crayfits_1    | [2022-03-23 22:35:52] php.CRITICAL: Uncaught Notice: Undefined variable: response {"exception":"[object] (ErrorException(code: 0): Notice: Undefined variable: response at /var/www/crayfits/src/Controller/FitsController.php:62)"} []
```

To test:
------
```
git clone  https://github.com/Islandora-Devops/isle-dc.git && cd $_
git checkout  issue-186-remove-unnecessary-reverse-proxy
alter sample.env: TAG=1.0.0-alpha-14
make demo
Then add a resource of type `moving image` and model  `video` with and attached media of type `video`
wait
check https://islandora.traefik.me/admin/content/media for a media of type `fits`
expect to be error free: `docker-compose logs --tail 100 crayfits` 
```

Proposal:
* tweak buildkit default `CRAYFITS_WEBSERVICE_URI=fits:8080/fits/examine`

**Comment by jefferya** [comment]
Added a proposed fix in [bceb9e9](https://github.com/Islandora-Devops/isle-buildkit/commit/bceb9e90b977e560c342e1215d830964ed8e73ff) that aims to prevent isle-dc users from having to add `CRAYFITS_WEBSERVICE_URI`

**Comment by nigelgbanks** [comment]
I've raised a pull request with your commit and merged it, good catch!
